incubator/java-openliberty/templates/default/src/test/java/it/dev/appsody/starter/EndpointTest.java

```diff
@@ -0,0 +1,84 @@
+package it.dev.appsody.starter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import javax.json.JsonObject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Response;
+
+import org.apache.cxf.jaxrs.provider.jsrjsonp.JsrJsonpProvider;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class EndpointTest {
+    
+    private static String baseUrl;
+    private static final String RESOURCE_ENDPOINT = "/starter/resource";
+    private Client client;
+    private Response response;
+    
+    @BeforeAll
+    public static void oneTimeSetup() {
+        String port = System.getProperty("liberty.test.port");
```

Comment:
This property matches the pom.xml config, of course.. might be worth a comment pointing that out.

Reply:
👍